### PR TITLE
Milestonone

### PR DESCRIPTION
**Pull request by hiromataba**
In this milestone, we have set up the project with webpack and build the list structure

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  const text = `<li class="item">
+                       <i class="uil uil-square circle-icon" job="complete"></i>
+                       <p class="text" > ${toDo}<p>
+                       <i class="uil uil-trash-alt" job="delete"></i>
+               </li>`;
+
+  const position = 'beforeend';
+  list.insertAdjacentHTML(position, text);
+}
+const arr = [];
```
Comment (Davidosky007) [inline src/index.js]: * I suggest you make an array of some simple to do tasks (array of objects) as it is a project requirement. Each task object should contain three keys eg [description [string], completed [bool], index: [number].

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+let index = 0;
+
+document.addEventListener('keyup', (event) => {
```
Comment (Davidosky007) [inline src/index.js]: * On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task. currently, your project lacks this functionality kindly fix this 🙏

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @hiromataba, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
CR3_gif_almost_there


## Required Changes ♻️

# _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by carlos-ssh — APPROVED** [review]
Hi @hiromataba, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**